core-bundle/src/Resources/contao/elements/ContentDownloads.php

```diff
@@ -77,7 +77,7 @@ public function generate()
 			{
 				if ($file == $this->objFiles->path || \dirname($file) == $this->objFiles->path)
 				{
-					\Controller::sendFileToBrowser($file);
+					\Controller::sendFileToBrowser($file, $this->disposition);
```

Comment:
You will need a fallback here for already existing elements in which case `$this->disposition` will be `null`.

Reply:
Same here, see above.